common/lib/xmodule/xmodule/course_module.py

```diff
@@ -634,6 +636,20 @@ class CourseFields(object):
                               default=False,
                               scope=Scope.settings)
 
+    license = License(
+        display_name=_("License"),
+        help=_("Select the license for this course. Reserve all rights, some rights, or no rights."),
+        default=None,
+        scope=Scope.settings
+    )
+
+    licensable = Boolean(
+        display_name=_("Licensable"),
+        help=_("Whether this course and its contents can be licensed using Creative Commons Licensing."),
+        default=hasattr(settings, 'FEATURES') and settings.FEATURES.get("DEFAULT_COURSE_LICENSABLE", False),
```

Comment:
I think we can safely assume that there will always be a `FEATURES` dict in the settings object. We assume that in many other places.